### DISCUSSION TITLE
System Bridge AppImage Bazzite Connection Refused

### DISCUSSION THREAD

**Discussion by SimpleHeuristics** [timmo001/system-bridge#3311] [Help]
Relatively new to Linux but was getting fed up with Windows.

I am now running Bazzite 41 (Fedora 41 based, immutable)

Just installed System Bridge 4.1.13 using the amd64 AppImage using GearLever.

When I launch the application it appears in the system tray and context menus work. However when I try to access any of the context menus items nothing works.

When I try to view data or change settings I'm simply met with a window that says: "Could not connect to 127.0.0.1: Connection refused"

I've tried to look in the KDE firewall settings and can't find any mention of systembridge.

I am also unable to run systembridge via the terminal using "systembridge"

I am not sure what next to troubleshoot.

**Comment by zany130** [comment]
Did you ever get this solved?

**Reply by SimpleHeuristics** [reply]
No, I've just been using another method to get the end result I needed which was just to turn my TV on after the PC boots!

**Comment by J-eremy** [comment]
#3370 is the exact same problem except Im on arch, used the appimage also and installed via gearlever.
